Make use of the flexibility of online meetings and schedule some impo… (#12)
mirjak/draft-shmoo-online-meeting: draft-kuehlewind-shmoo-online-meeting.md

to rotate between Asia, Europe,  and North America. While the exact time slot used had let to various discussions,
following this 1-1-1 rule to share the pain has/seems to have rough consensus.

## Full vs. limited agenda {#full-limited}

The IETF-108 meeting survey asked about the structure of that meeting (full meeting) compared to that of IETF 107,
which hosted only a limited set of session followed by interims in the weeks after. The structure of IETF 108 was
preferred by 82% [https://www.ietf.org/blog/ietf-108-meeting-survey/]. While the limited agenda of IETF-107
could have been a good one-time replacement, the value of cross-participation and high active meetings weeks
has been recognised as important for continuous progress (and not only for newly initiated work).

> 

Reviewer: I think a fuller discussion of pros and cons would be valuable here.

Interim meetings:
- reduce scheduling conflicts during IETF week
- are less likely to attract people with casual interest
- are optimally scheduled for the key participants
- are not scheduled to occasionally support people in less-common time zones

a saving of in total 10-20 minutes per day might not balance the need to use the breaks for recreation or 
at least some socialising.

## Number of Parallel Tracks {#tracks}

Fully online meetings are not limited in the number of parallel tracks by the physical restriction of a meeting venue aka

> the number of meeting rooms. In order to reduce the number of possible conflicts, it is anyway

Author: ```suggestion
the number of meeting rooms. In order to reduce the number of possible conflicts, it is still
```

desirable to minimise the number of parallel tracks by balancing the requested sessions mostly equally over the available slots.

> Still, if the total number of requested sessions exceeds the capacity of the usual 8 parallel tracks, it

Author: ```suggestion
If the total number of requested sessions exceeds the capacity of the usual 8 parallel tracks, it
```

> If the total number of requested sessions exceeds the capacity of the usual 8 parallel tracks, it
> is possible for a fuly online meeting to simply use more tracks. This also means, if the number of meeting days is seen as fixed, this decision

Author: ```suggestion
But if the total number of requested sessions exceeds the capacity of the usual 8 parallel tracks, it
is possible for a fully online meeting to simply use more tracks. This also means, if the number of meeting days is seen as fixed, this decision
```

is implicitly made by the working group chairs requesting a certain number of sessions and length. 

As more parallel sessions usually also mean more conflicts,

> chairs are encouraged to request plenary meeting time carefully but also based realistic planning to avoid running over time.

Author: ```suggestion
chairs are encouraged to request plenary meeting time carefully but also based on realistic planning to avoid running over time.
```

If the number of meeting days is seen as fixed, this decision is implicitly made by the working group
chairs requesting a certain number of sessions and length. While realistic planning is desired to avoid running over time,
chairs are encouraged to request plenary meeting time carefully and use interims where possible and sensible
instead.

However, online meetings also provide an opportunity to use more time more flexibly. While for an in-person meeting all
sessions have to be fitted into the available time people are willing to travel at once (usually roughly a week), online meetings do not have
that constraint. Therefore there is a trade-off between the number of parallel tracks, where more parallel tracks
mean more potential conflicts (as least of high-active participants), and the overall time in terms of hours per day or 
total days used. 

A highly concentrated meeting, in structure similar to the in-person plenary meeting, provides value for cross-participants
(see also section {{full-limited}}). Further a well defined meeting time, rather than spreading many interims over the whole

> year can make deconflicting with other non-IETF meetings easier. Still opportunities to utilize the flexibility a fully only meeting

Reviewer: I would move this discussion of the value of a concentrated meeting into {#full-limited}

> year can make deconflicting with other non-IETF meetings easier. Still opportunities to utilize flexibility of a fully only meeting
> provide to reduce conflicts during the plenary meeting should be further explored. E.g. it would be possible to keep most regular

Author: ```suggestion
year can make deconflicting with other non-IETF meetings easier. Still opportunities to utilize the flexibility a fully only meeting
provides to reduce conflicts during the plenary meeting should be further explored. E.g. it would be possible to keep most regular
```

working group sessions within the usually five main meeting days but have some of the more conflicted sessions such as BoFs or
dispatch meetings in other dedicated time slots. As the Hackathon for online only meetings is

> usually held in the week before the online plenary meeting {{I-D:ietf-shmoo-hackathon}}, that week is already a high active

Author: ```suggestion
usually held in the week before the online plenary meeting {{?I-D.ietf-shmoo-hackathon}}, that week is already a high active
```
